### PR TITLE
JavaScript: Fix formatting on long named exports

### LINKED ISSUE #4729: Import and export formatted with different logic
**Prettier 1.13.5**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBLAtgBwgJxgAmHwBM4AzAQwFcAbAigZ3zIlbfY869fwF9mcEdPgDkLbhMmsRAHShwAHtjyES5anXyNmU3Vz4Cho8XtMQRIADQgImGKmgNkoCjkEB3AAquETlBQA3CFRiKxAAIxwKMABrOBgAZUxo1CgAc2QYHCo4a1SGODxPKLT0CmRKGgLrACsGBQAhKNj4hIp0OAAZVLgKiirckGScApxkCIpwgE8aaDDMHFSYAHUQmAALZAAOAAZrBYgC5ajMcYW4UYDe6xw4AEcqVFviilLypErqkAL0VEzswYMVJpGhwACKVAg8D6A2sMEmq2IG2QACY4VFUDRgQBhIRlcZQaDXEBUAoAFUmfk+cF4vCAA)
```sh
--parser babylon
```

**Input:**
```jsx
import { default as foooooooooooooooooooooooooooooo } from 'foooooooooooooooooooooooooooooo'
export { default as foooooooooooooooooooooooooooooo } from 'foooooooooooooooooooooooooooooo'
```

**Output:**
```jsx
import { default as foooooooooooooooooooooooooooooo } from "foooooooooooooooooooooooooooooo";
export {
  default as foooooooooooooooooooooooooooooo
} from "foooooooooooooooooooooooooooooo";

```

**Expected behavior:**

I expect that import and exports are printed the same. AFAIK the format of the import is intended so I would like if the export would align with that. The benefit it provides is that you can sort export statements a lot quicker if they are in a single line.

Also it looks a lot cleaner if you have a file which basically only reexports other modules.

### PR DESCRIPTION
**Pull request by sosukesuzuki**
Fix #4729

- [x] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [x] (If the change is user-facing) I’ve added my changes to the `CHANGELOG.unreleased.md` file following the template.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: src/language-js/printer-estree.js [hunk 1]
```diff
@@ -4591,28 +4591,46 @@ function printExportDeclaration(path, options, print) {
...
-                    join(concat([",", line]), specifiers)
-                  ])
-                ),
-                ifBreak(shouldPrintComma(options) ? "," : ""),
-                options.bracketSpacing ? line : softline,
-                "}"
-              ])
-            )
-          : ""
+        content
```
Comment (evilebottnawi) [inline src/language-js/printer-estree.js]: `content` -> `printed`

File: src/language-js/printer-estree.js [hunk 2]
```diff
@@ -4591,28 +4591,46 @@ function printExportDeclaration(path, options, print) {
         defaultSpecifiers.length !== 0 &&
         (namespaceSpecifiers.length !== 0 || specifiers.length !== 0);
 
+      let content = "";
+      if (specifiers.length !== 0) {
+        if (
+          specifiers.length === 1 &&
+          defaultSpecifiers.length === 0 &&
+          decl.specifiers &&
+          !decl.specifiers.some(node => node.comments)
```
Comment (evilebottnawi) [inline src/language-js/printer-estree.js]: Maybe we can move this stuff to `canBreak` helper?

**Comment by sosukesuzuki** [comment]
@evilebottnawi Thanks reviewing, I fixed.

**Comment by evilebottnawi** [comment]
Need fix lint CI

**Comment by sosukesuzuki** [comment]
I fixed conflicts

**Comment by lipis** [comment]
How does it work with two long named imports?

**Comment by sosukesuzuki** [comment]
@lipis 
Work like below (also, I added tests)
[Playground link](https://deploy-preview-6446--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACYPAMwlLPIsqupvIBpjanmXy88BfY7CAWzwA6IEq1ZCA3AKgg6ICBhgBLaAGdkoAIbYeAdwAKWhGpQaAbhEUATGSABG2DWADWcGAGUMjxVADmyGNgArnCyABYwvAA2AOqhivAqnmBwbkbxiqbxAJ7I4Cpqst4qcLh6Dj68GshEGpHFsgBWKmgAQg7Orm4avHAAMt5w1bX1IE1obt4+kXAAioEQ8EN1ISCe2MXYubYatnCRNhjY3jDRVjChyAAcAAyyhxDF0Q4YuYdwG6aDsgCO8-Bl8mMIA0KgAtFA4HBLFCbNg4L9FHCyhoKlUkDVlrJirxFP4gisVJNpnMFoN0cMVjAdqdLOdkAAmWQBDSKSKTADCfEquSg0C+IECxQAKjtjBiRqZggBJKDQ2BuMBHBQAQVlbhgWWmS2KHA4QA)

**Input**
```js
export { fooooooooooooooooooooooooooooooooooooooooooooooooo, foooooooooooooooooooooooooooooooooooooooooooooooooo  } from "fooooooooooooooooooooooooooooo";
```

**Output**
```js
export {
  fooooooooooooooooooooooooooooooooooooooooooooooooo,
  foooooooooooooooooooooooooooooooooooooooooooooooooo
} from "fooooooooooooooooooooooooooooo";
```

**Comment by lipis** [comment]
But let's wait for one more.

**Comment by sosukesuzuki** [comment]
@evilebottnawi I fixed conflicts

**Pull request merged**